src/palace/manager/core/opds_import.py

```diff
@@ -425,9 +425,9 @@ def parse_identifier(self, identifier: str | None) -> Identifier | None:
             result = Identifier.parse_urn(self._db, identifier)
             if result is not None:
                 parsed_identifier, _ = result
-        except Exception:
-            self.log.error(
-                f"An unexpected exception occurred during parsing identifier {identifier}"
+        except Exception as e:
+            self.log.exception(
+                f"An unexpected exception occurred during parsing identifier {identifier}: {e}"
```

Comment:
Very minor: Since we're touching this, could we quote the identifier, so that it doesn't blend in with the rest of the message?

```
f"An unexpected exception occurred during parsing identifier '{identifier}': {e}"
```